internal/cmd/stack/environment.go

```diff
@@ -73,6 +117,132 @@ func setVar(cliCtx *cli.Context) error {
 	return nil
 }
 
+func (e *listEnvCommand) listEnv(cliCtx *cli.Context) error {
+	outputFormat, err := cmd.GetOutputFormat(cliCtx)
+	if err != nil {
+		return err
+	}
+
+	stackID := cliCtx.String(flagStackID.Name)
+
+	var query listEnvQuery
+	variables := map[string]interface{}{
+		"stack": graphql.ID(stackID),
+	}
+
+	if err := authenticated.Client.Query(cliCtx.Context, &query, variables); err != nil {
+		return err
+	}
+
+	var elements []listEnvElementOutput
+	for _, config := range query.Stack.RuntimeConfig {
+		var contextName *string
+		if config.Context != nil {
+			contextName = &config.Context.ContextName
+		}
+		if element, err := config.Element.toConfigElementOutput(contextName); err == nil {
+			elements = append(elements, element)
+		} else {
+			return err
+		}
+	}
+
+	switch outputFormat {
+	case cmd.OutputFormatTable:
+		return e.showOutputsTable(elements)
+	case cmd.OutputFormatJSON:
+		return e.showOutputsJSON(elements)
+	default:
+		return fmt.Errorf("unknown output format: %v", outputFormat)
+	}
+}
+
+func (e *listEnvCommand) showOutputsTable(outputs []listEnvElementOutput) error {
+	tableData := [][]string{{"Name", "Type", "Value", "File Mode", "Context"}}
+	for _, output := range outputs {
+		var row []string
+
+		row = append(row, output.Name)
+		row = append(row, output.Type)
+
+		var value string
+		switch {
+		case output.Runtime:
+			value = "*****"
+		case output.WriteOnly:
+			value = "<computed>"
```

Comment:
These two are flipped - `Runtime` means the value is calculated at runtime (i.e. `<computed>`), and `WriteOnly` means it's a secret.

Reply:
I see, the values are now switched.